File: lab2/Lab_2_3.py
num = int(input("Enter an integer between(inclusive) 3 and 9: "))
for i in range(num+2):
    for j in range(1, i):
        print(j, end="")
    print("")
for k in range(num, 0 , -1):
    for j in range(1, k):
        print(j , end="")
    print("")

File: lab2/Lab_2_3.cpp
#include <iostream>

using namespace std;

int main(){
    cout << "Enter an integer between(inclusive) 3 and 9: ";
    int num;
    cin >> num;

    for(int i = 0; i<num+2; i++){
        for(int j = 1; j<i; j++){
            cout << j;
        }
        cout << endl;
    }
    for(int i = num-1; i >= 0; i--){
        for(int j = 1; j < i; j++){
            cout << j;
        }
        cout << endl;
    }
    return 0;
}